person.rb

```diff
@@ -1,15 +1,16 @@
-require './corrector.rb'
+require './corrector'
 
 class Person
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id, :corrector
+  attr_reader :id, :corrector, :rentals
```

Comment:
- [ ] Getters and Setters for rentals should be present, please do so to comply with the project requirements (add it to the accessor part of this file)

Reply:
Thank you for your valuable comment. I changed it as you can see in the image below:
![personattr](https://user-images.githubusercontent.com/60292761/141265867-0cf8c960-7bb3-42d0-a517-67013d36259f.PNG)